examples/asr/speech_to_text.py

```diff
@@ -29,6 +30,7 @@
 
 from nemo.collections.asr.arguments import add_asr_args
 from nemo.collections.asr.models import EncDecCTCModel
+from nemo.core.classes.optimizers import add_optimizer_args
```

Comment:
I think "optimizers" should be just under nemo.core.optimizers. Maybe rename that to nemo.core.optim since we can put LR policies there as well?